eng/vscode/README.md

```diff
@@ -75,43 +75,297 @@ You’re all set! Azure MCP Server is now ready to help you work smarter with Az
 Here are some cool prompts you can try across our supported Azure services:
 
 ### 🔎 Azure AI Search
+
 * "What indexes do I have in my Azure AI Search service 'mysvc'?"
 * "Let's search this index for 'my search query'"
 
 ### ⚙️ Azure App Configuration
+
 * "List my App Configuration stores"
 * "Show my key-value pairs in App Config"
 
 ### 📦 Azure Container Registry (ACR)
+
 * "List all my Azure Container Registries"
 * "Show me my container registries in the 'myproject' resource group"
+* "List all my Azure Container Registry repositories"
 
 ### ☸️ Azure Kubernetes Service (AKS)
+
 * "List my AKS clusters in my subscription"
 * "Show me all my Azure Kubernetes Service clusters"
 
 ### 📊 Azure Cosmos DB
+
 * "Show me all my Cosmos DB databases"
 * "List containers in my Cosmos DB database"
 
+### 🧮 Azure Data Explorer
+
+* "Get Azure Data Explorer databases in cluster 'mycluster'"
+* "Sample 10 rows from table 'StormEvents' in Azure Data Explorer database 'db1'"
+
+### ⚡ Azure Managed Lustre
+
+* "List the Azure Managed Lustre clusters in resource group 'my-resourcegroup'"
+* "How many IP Addresses I need to create a 128 TiB cluster of AMLFS 500?"
+
+### 📊 Azure Monitor
+
+* "Query my Log Analytics workspace"
+
+### 🔧 Azure Resource Management
+
+* "List my resource groups"
+* "List my Azure CDN endpoints"
+* "Help me build an Azure application using Node.js"
+
 ### 🗄️ Azure SQL Database
+
 * "Show me details about my Azure SQL database 'mydb'"
 * "List all databases in my Azure SQL server 'myserver'"
+* "List all firewall rules for my Azure SQL server 'myserver'"
+* "List all elastic pools in my Azure SQL server 'myserver'"
+* "List Active Directory administrators for my Azure SQL server 'myserver'"
 
 ### 💾 Azure Storage
+
 * "List my Azure storage accounts"
 * "Get details about my storage account 'mystorageaccount'"
-* "Create a new storage account with premium SSD storage"
+* "Create a new storage account in East US with Data Lake support"
 * "Show me the tables in my Storage account"
 * "Get details about my Storage container"
+* "Upload my file to the blob container"
+* "List paths in my Data Lake file system"
+* "List files and directories in my File Share"
+* "Send a message to my storage queue"
+
+## 🛠️ Currently Supported Tools
+
+<details>
+<summary>The Azure MCP Server provides tools for interacting with the following Azure services</summary>
```

Comment:
This will cause the data for rest of the document to be collapsed in a drop down and not visible by default.

Reply:
fixed it